drivers/vulkan/vulkan_context.cpp

```diff
@@ -2018,8 +2002,8 @@ Error VulkanContext::prepare_buffers() {
 	vkWaitForFences(device, 1, &fences[frame_index], VK_TRUE, UINT64_MAX);
 	vkResetFences(device, 1, &fences[frame_index]);
 
-	for (KeyValue<int, Window> &E : windows) {
-		Window *w = &E.value;
+	for (KeyValue<int, Window> &e : windows) {
```

Comment:
We use uppercase E, F, etc. when iterating Godot containers that return indirect objects instead of the actual elements.